jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-client/src/test/java/org/jbpm/workbench/cm/client/newcase/NewCaseInstancePresenterTest.java

```diff
@@ -71,7 +77,7 @@ public void init() {
 
     @Test
     public void testCreateInvalidCaseInstance() {
-        presenter.createCaseInstance(null);
+        presenter.createCaseInstance(null, null);
```

Comment:
How about putting anyString() call as the second argument instead of null, thus making it clear that the value of the second argument is irrelevant in this test?